zap/src/main/java/org/parosproxy/paros/extension/history/PopupMenuExportMessage.java

```diff
@@ -55,7 +56,9 @@
 import org.parosproxy.paros.network.HttpMessage;
 import org.zaproxy.zap.view.widgets.WritableFileChooser;
 
-public class PopupMenuExportMessage extends JMenuItem {
+/** @deprecated in version 2.12.0, see the exim add-on */
+@Deprecated
+class PopupMenuExportMessage extends JMenuItem {
```

Comment:
Accessibility is being changed, and causing the error mentioned in https://github.com/zaproxy/zaproxy/issues/6579#issuecomment-1125967077

Reply:
Oops, thanks good catch!